src/main/java/edu/harvard/iq/dataverse/engine/command/impl/UpdateDatasetVersionCommand.java

```diff
@@ -102,7 +103,10 @@ public Dataset execute(CommandContext ctxt) throws CommandException {
         }
         
         Dataset theDataset = getDataset();        
-        ctxt.permissions().checkUpdateDatasetVersionLock(theDataset, getRequest(), this);
+        //ctxt.permissions().checkUpdateDatasetVersionLock(theDataset, getRequest(), this);
+        // this is an experiment (probably temporary)
```

Comment:
Yes, I meant to reverse (and re-test) this "experiment". The idea was to do a fresh transaction database lookup for any locks that may have been added since the dataset instance was instantiated... I don't think it's needed after all.

Reply:
I thought about that, but, I chose to revert the command to its previous state. The Globus code should continue working at least as well as before. And I am hesitant to change the method in the PermissionServiceBean, for fear of some exotic scenario where something that's working now is going to break.